template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/assets/ml-artifacts-asset.yml.tmpl

```diff
@@ -1,56 +1,3 @@
-# Deployment target specific values
-targets:
-  dev:
-    resources:
-      {{ if (eq .input_include_models_in_unity_catalog `no`) }}models:
-        model:
-          description: MLflow registered model for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.
-      {{- else -}}
-        registered_models:
-        model:
-          comment: Registered model in Unity Catalog for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.{{end}}
-
-  test:
-    resources:
-      {{ if (eq .input_include_models_in_unity_catalog `no`) }}models:
-        model:
-          description: MLflow registered model for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.
-      {{- else -}}
-        registered_models:
-        model:
-          comment: Registered model in Unity Catalog for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.{{end}}
-
-  staging:
-    resources:
-      {{ if (eq .input_include_models_in_unity_catalog `no`) }}models:
-        model:
-          description: MLflow registered model for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.
-      {{- else -}}
-        registered_models:
-        model:
-          comment: Registered model in Unity Catalog for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.{{end}}
-
-  prod:
-    resources:
-      {{ if (eq .input_include_models_in_unity_catalog `no`) }}models:
-        model:
-          description: |
-            MLflow registered model for the "{{template `project_name` .}}" ML Project. See the corresponding [Git repo]($#{var.git_repo_url}) for details on the project.
-
-            Links:
```

Comment:
Removing these links since `var.git_repo_url` has been broken for a while, and linking to these jobs does not merit providing `databricks_prod_workspace_host` as a variable.